### ISSUE TITLE
Cannot read property 'mq' of undefined

### ISSUE THREAD

**Issue by maddoxnelson** [pedronauck/docz#452]
## Bug Report

On starting a new project with no dependencies other than `docz`, I am unable to recognize any .mdx files.

**To Reproduce**

Set up:
1. `mkdir <your-docz-project> && cd <your-docz-project>`
1. `npm init`
1. Add `"docz:dev": "docz dev"` to your scripts
1. `touch any-name.mdx`
1. Put some text in the .mdx file. I used the markup from the Getting Started guide: 
```---
name: Hello world
---

# Hello world

Hello, I'm a mdx file!
```

1. `npm run docz:dev`
1. I wind up with this: 
![pic 2018-11-15 at 3 44 05 pm](https://user-images.githubusercontent.com/4610199/48580470-4f4ade80-e8ed-11e8-8ae9-155f4dd0c652.png)

**Here's a test Repository:* https://github.com/maddoxnelson/new-docz

**Expected behavior**

Docz should return my stuff, as described in theGetting Started page.

**Environment**

* OS: OSX 10.13.6 (High Sierra)
* Node/npm version: Node: 10.12.0, NPM: 6.4.1

**Comment by magicspon** [comment]
I'm also seeing this

**Comment by idrakimuhamad** [comment]
Seeing this too as I JUST created one an hour ago. Probably v0.12.11 having a problem.

**Comment by maddoxnelson** [comment]
(Not sure if this has anything to do with it) I do **not** have yarn installed globally -- I can't actually use it for work-related reasons. I tried to spin up docz with `npm run bs` and I noticed there seems to be a dependency on yarn being installed globally:

This is what I get when I run `npm run bs`:
```

lerna notice cli v3.4.3
lerna WARN ECYCLE Dependency cycles detected, you should fix these!
lerna WARN ECYCLE docz-example-basic -> docz -> docz-theme-default -> docz -> docz@^0.12.11
lerna WARN ECYCLE docz-example-flow -> docz -> docz-theme-default -> docz -> docz@^0.12.11
lerna WARN ECYCLE docz-example-styled-components -> docz -> docz-theme-default -> docz -> docz@^0.12.11
lerna WARN ECYCLE docz-example-typescript -> docz -> docz-theme-default -> docz -> docz@^0.12.11
lerna WARN ECYCLE docz-theme-default -> docz -> docz-theme-default
lerna WARN ECYCLE docz -> docz-theme-default -> docz
lerna info bootstrap root only
lerna ERR! yarn install --mutex network:42424 --non-interactive exited ENOENT in 'docz'
lerna ERR! yarn install --mutex network:42424 --non-interactive exited ENOENT in 'docz'

```

**Comment by popugang** [comment]
same problem

**Comment by Jayphen** [comment]
Me too

**Comment by pedronauck** [comment]
Thanks to report guys, I'm releasing a patch now with this fix 🙏

**Comment by pedronauck** [comment]
Fixed on release [v0.12.12](https://github.com/pedronauck/docz/releases/tag/v0.12.12) ✅
